src/main/java/amazin/controller/ApplicationController.java

```diff
@@ -15,27 +15,27 @@
 
     @GetMapping({ "/" })
     public String index(Model model) {
-        model.addAttribute("books", bookRepository.findAll());
+        model.addAttribute("book", bookRepository.findAll());
         return "index";
     }
 
-    @GetMapping({"/login"})
-    public String login(){
+    @GetMapping({ "/login" })
+    public String login() {
         return "login";
     }
 
-    @GetMapping({"/register"})
-         public String register(){
+    @GetMapping({ "/register" })
```

Comment:
this is not okay

Reply:
this is done automatically by my linter, can't do anything about it